code/app/Http/Controllers/LodestoneController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Thaliak\Http\Controllers;
+
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
+use Illuminate\Http\Request;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Storage;
+use Thaliak\Http\Controllers\Controller;
+use Thaliak\Http\Lodestone\Api;
+use Thaliak\Http\Lodestone\Character as Character;
+use Thaliak\Http\Lodestone\FreeCompany as FreeCompany;
+use Thaliak\Models\World;
+
+class LodestoneController extends Controller
+{
+    protected $lodestone; // Api
+
+    public function __construct(Api $lodestone)
+    {
+        $this->middleware('api-auth');
```

Comment:
Needs to be `auth:api` (`api-auth` was the name of a middleware group I was using before I changed the auth workflow).